src/pubsub.js

```diff
@@ -197,16 +161,42 @@ class BasicPubSub extends Pubsub {
           return
         }
 
-        this._processRpcMessage(msg)
+        this._processRpcMessage(peer, msg)
       })
     }
-    this._handleRpcControl(peer, rpc)
   }
 
   /**
-   * @param {rpc.RPC.Message} msg
+   * Handles an subscription change from a peer
+   *
+   * @param {Peer} peer
+   * @param {RPC.SubOpt} subOpt
    */
-  _processRpcMessage (msg) {
+  _processRpcSubOpt (peer, subOpt) {
```

Comment:
We should probably make this name more clear and not abbreviate. What do you think?

Reply:
Agreed its not a super helpful name. The naming of this, `_processRpc`, `_processRpcMessage`, and `_processRpcControlMessage` were meant to align with the names in the protobuf definition. ie `RPC`, `SubOpt`, `Message`, and `ControlMessage`. Definitely interested in other ideas for consistent naming

Reply:
I was thinking about expanding the `SubOpt`, but you make a good point. I think that it is a good call to align the namings with the protoBuf definition, rather than make an exception here.